package.json

```diff
@@ -10,7 +10,8 @@
     "lint": "yarn lint:sol && yarn lint:ts",
     "lint:sol": "solhint 'src/contracts/**/*.sol'",
     "lint:ts": "eslint --max-warnings 0 .",
-    "fmt:sol": "prettier 'src/contracts/**/*.sol' -w"
+    "fmt:sol": "prettier 'src/contracts/**/*.sol' -w",
+    "prepack": "yarn build && yarn deploy --network rinkeby"
```

Comment:
I'm not sure we should deploy on `prepack`. This means we will have a new rinkeby deployment for every packaged version even if only TS changes for example.

Reply:
I don't think so. Since deployments are deterministic based on bytecode this will be a noop if the contract code didn't change.

Reply:
:exploding_head: same as above.